Formula/neomutt.rb

```diff
@@ -0,0 +1,37 @@
+class Neomutt < Formula
+  desc "Email reader with support for Notmuch (fast indexing), News (NNTP) and much more"
+  homepage "https://www.neomutt.org/"
+  url "https://github.com/neomutt/neomutt/archive/neomutt-20171215.tar.gz"
+  sha256 "7fb76e99a9f23715ad772ad8f7008c6e2db05eed344817055176c76dbd60c1b5"
+  head "https://github.com/neomutt/neomutt.git"
+
+  depends_on "gettext" => :build
```

Comment:
In the prior run, this was shown not to be build-time only due to
```
dyld: Library not loaded: /usr/local/opt/gettext/lib/libintl.8.dylib
  Referenced from: /usr/local/Cellar/neomutt/20171215/bin/neomutt
  Reason: image not found
```
so the `=> :build` is erroneous unless you can block the libintl linkage with a configure option (probably `--disable-nls`). Not sure whether you actually want nls on or off

Reply:
@flatcap what about nls support. Do you want nls to be enabled or disabled?